tests/manager/manager-debug-001.phpt

```diff
@@ -29,7 +29,7 @@ foreach($content as $line) {
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-[%s]     PHONGO: DEBUG   > Creating Manager, phongo-0.%d.%d[%s] - mongoc-1.%s, libbson-1.%s
+[%s]     PHONGO: DEBUG   > Creating Manager, phongo-%d.%d.%d[%s] - mongoc-1.%s, libbson-1.%s
```

Comment:
@bjori: Should I also rename "phongo", or do we expect this to differ between HHVM and PHP? AFAIK, we're only using "phongo" in the source filenames. This may be one of the few places it's still presented publicly.